src/library/Uno.Material/Styles/Controls/Button.xaml

```diff
@@ -424,4 +435,272 @@
 		</Setter>
 	</Style>
 
+	<!-- Issue #41: Todo remove not_wasm when fix for Ellipse Fill with SolidColorBrushes that contain opacity is merged -->
```

Comment:
I noticed this style isn't shown in the samples, we have 3 styles but only 2 samples.